src/effects/imagefilters/SkBlurImageFilter.cpp

```diff
@@ -173,29 +179,39 @@ class Pass {
         // Drain the remaining blur values into dst assuming 0's for the leading edge.
         if (dstIdx < dstEnd) {
             int n = dstEnd - dstIdx;
-            this->blurSegment(n, nullptr, 0, dstCursor, dstStride);
+            
+            uint32_t trailingEdge = 0;
+            if (fTileMode == SkTileMode::kMirror) {
+                // Go back by 2 in order to not sample the anti-aliased edge
+                trailingEdge = *(srcCursor - 2 * srcStride);
```

Comment:
Is this guaranteed to be valid? Even for edge-cases like 1x1 images or is there a min- width/height for this to work?

Reply:
This wasn't correct anyway. I've now fixed it for a proper "mirrored" tile mode by going backwards through the pixels and sampling correctly instead of just repeating the last one.